src/frontend/classic/components/FolderSelectorInput.svelte

```diff
@@ -0,0 +1,44 @@
+<script lang="ts">
+    import { TextInput } from "carbon-components-svelte";
+    import { onSettingChange } from "../utils/storage";
+    import type { SettingExtras, SettingValidator } from "../utils/storage";
+    import { v4 as uuidv4 } from "uuid";
+
+    export let labelText = "Select directory";
+    export let storageKey: string | undefined = undefined;
+    export let extras: SettingExtras = undefined;
+    export let validator: SettingValidator = undefined;
+    export let passNewValueToExtras: boolean = false;
+    export let path: string = "";
+
+    const uuid = uuidv4();
```

Comment:
Yes we can pass a selfdefiningID as a prop. The idea was to abstract this logic inside the component, so we don't have to handle this when calling the FolderSelectorInput component. But yeah, it might be overkill, and adds an extra dependency. I will fix this